editoast/src/core/stdcm.rs

```diff
@@ -55,7 +56,9 @@ pub struct STDCMRequest {
     /// Margin to apply to the whole train
     pub margin: Option<MarginValue>,
     /// List of planned work schedules
-    pub work_schedules: Vec<LightWorkSchedule>,
+    pub work_schedules: Vec<WorkSchedule>,
+    // List of applicable temporary speed limits between the train departure and arrival
```

Comment:
You can do a small check: `git diff dev | grep '[^/]//[^/]'`.

```suggestion
    /// List of applicable temporary speed limits between the train departure and arrival
```